database/deltas/execute.go

```diff
@@ -46,15 +46,18 @@ func setMultiStatement(connString string) string {
 
```

Comment:
this file can be reverted to just use the global. The log statements largely debug in nature with only a couple of exceptions where the error that is being logged is also passed back. We can initialize the database in the initializer to just be the global one as well.